.gitignore

```diff
@@ -6,3 +6,4 @@ vendor/
 .DS_Store
 go.sum
 localenvoyconfig.yaml
+localdev/
```

Comment:
Thanks a lot!
TBH I didn't expect the `localdev/` at `.gitignore`, that's a folder that is useful to me, but probably some other people don't use it, or use a different name.

What about adding a comment above it to clarify it's purpose?
```

# Developer's personal hack files
localdev/
```
What about adding a comment above it to indicate the purpose?

Also, should `localenvoyconfig.yaml` be removed and the `Makefile` adapted to:

https://github.com/heptio/contour/blob/5046e28f053d1ff88e174fe4593a89bcd0c2ce0e/Makefile#L6


```
LOCAL_BOOTSTRAP_CONFIG = localdev/envoyconfig.yaml
```

Reply:
Where does `localdev` come from? I don't see it referenced anywhere other than the .gitignore.

Reply:
if was a _light request_ here https://github.com/heptio/contour/issues/1191

I find it handy having my local envs and scripts at some folder inside the repo.
I saw that you are already using `localenvoyconfig.yaml`, the `localdev/` is a proposal to put in there any local dev assets, that might include the envoy config.

Reply:
That's fine, but I don't see `localdev` being used in this PR. It's referenced in the .gitignore but nowhere else (unless I'm missing it).

Reply:
oh I see!

Yes, the PR is not using it, `localdev` would be that folder where you put handy scripts that are useful only at your local box, env vars pointing to your container registry, cloud creds if needed ... there should be no reference  to it at code.

But then, back at the issue I didn't thought about moving the `localenvoyconfig.yaml` to that directory because I expected some back and forth regarding that entry at the `.gitignore` ... since that's part of the current PR, I think it would be ok ask for another commit here that would modify `Makefile` to look for that file inside that local folder instead of having it hanging at the root.

Not sure if I explained it ok, and if that makes sense.

I'm at the slack channel if you want to reach out

Reply:
Good question @stevesloka! Yes, `localdev/` is listed only in the .gitignore file, no where else. I really like @odacremolbap's suggestion about comments and moving the envoyconfig.yaml files in there, I will do that later today. Maybe `local/` may be a better name though.